### ISSUE TITLE
Adapt swisssearch to accept wider range of  input formats for coords

### ISSUE THREAD

**Issue by davidoesch** [geoadmin/mf-geoadmin3#4051]
based on our visit in Dubendorf and NAVREF
-> adapt swissearch to support the LW Dubendorf uses cases

LV03 and LV95
- [x] 600'000 200'000
- [x] 600000 200000
- [x] 600 000 200 000
- [x] 600000/200000 ( is high prio since  the official https://www.cadastre.ch/de/manual-av/service/checkservice.html is giving back this result
- [x] 600000 / 200000
- [x] 600000,200000
- [x] 600000 200000
- [x] 200000 600000

WGS84 (zoom to church of La Brévine)
- [x] 46.97984  6.60757    (DD)
- [x] 46°58.7904'  6°36.4542'  (DM)
- [x] 46°58'47.424''   6°36'27.252''  (DMS, double single quote)
- [x] 46°58'47.424"   6°36'27.252"    (DMS, double quote)  

- [x] Update  https://help.geo.admin.ch/?id=24

**Comment by oterral** [comment]
Which are?

**Comment by procrastinatio** [comment]
Any coordinates order, any kind of thousand separator and separator. Any order also. Something like
```
600'000 200'000
600000 200000
600 000 200 000
600000/200000
600000,200000
600000 200000
200000 600000
```
And for degrees
```
7.345
7°45'45.344"
7°45'45.344''
```

**Comment by davidoesch** [comment]
I asked Donat to provide a list

**Comment by loicgasser** [comment]
FYI
http://lubire.donatellis.ch/mobile.php#
The app!

**Comment by davidoesch** [comment]
I deleted your comment... Since he does not want the URL spread

**Comment by pauloamado** [comment]
Help is updated

**Comment by oterral** [comment]
thx

**Comment by AFoletti** [comment]
Skyguide noticed a regression after this merge and asks for a fix (customer feedback ID 2018051810592985)

Problem:
47°27'29.4944"N 008°32'52.8164"E  --> does not work
47°27'29.4944"N 08°32'52.8164"E --> works fine (notice the missing "0" in Easting!)

Expected:
47°27'29.4944"N 008°32'52.8164"E  works, since it's a perfectly valid coordinate pair

**Comment by procrastinatio** [comment]
Please update the file https://github.com/geoadmin/mf-geoadmin3/blob/master/test/specs/search/SearchService.spec.js with all test cases we have to support. We'll make them pass.
